amethyst_rendy/src/submodules/texture.rs

```diff
@@ -29,7 +29,7 @@ enum TextureState<B: Backend> {
 }
 
 #[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
-pub struct TextureId(u32);
+pub struct TextureId(pub u32);
```

Comment:
This wrapper type is done specifically to not allow constructing arbitrary texture ids out of thin air. The inner type should not be public

Reply:
Removed.